### ISSUE TITLE
Conflict between SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing and Squiz.ControlStructures.ControlSignature.SpaceAfterCloseParenthesis

### ISSUE THREAD

**Issue by morozov** [slevomat/coding-standard#1085]
```
$ composer show | grep -P 'coding-standard|php_codesniffer'

doctrine/coding-standard                       8.1.0  The Doctrine Coding Standard is a s...
slevomat/coding-standard                       6.4.0  Slevomat Coding Standard for PHP_Co...
squizlabs/php_codesniffer                      3.5.6  PHP_CodeSniffer tokenizes PHP, Java...
```
Consider the following code example:

```php
<?php if (true): ?>
    foo
<?php else : ?>
    bar
<?php endif; ?>
```
Try checking (note the “found -1”):
```
$ phpcs -s --standard=Doctrine \
    --sniffs=SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing,Squiz.ControlStructures.ControlSignature \
    test.php

FILE: /home/morozov/Projects/phpcs-playground/test.php
------------------------------------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 1 LINE
------------------------------------------------------------------------------------------------
 1 | ERROR | [x] Expected 1 lines after "if", found -1.
   |       |     (SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing.IncorrectLinesCountAfterControlStructure)
 1 | ERROR | [x] Expected 1 space(s) after closing parenthesis; found 0
   |       |     (Squiz.ControlStructures.ControlSignature.SpaceAfterCloseParenthesis)
------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
------------------------------------------------------------------------------------------------

Time: 63ms; Memory: 10MB
```
Try fo fix:
```
$ phpcbf \
    --standard=Doctrine \
    --sniffs=SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing,Squiz.ControlStructures.ControlSignature \
    test.php

PHPCBF RESULT SUMMARY
----------------------------------------------------------------------
FILE                                                  FIXED  REMAINING
----------------------------------------------------------------------
/home/morozov/Projects/phpcs-playground/test.php      FAILED TO FIX
----------------------------------------------------------------------
A TOTAL OF 0 ERRORS WERE FIXED IN 1 FILE
----------------------------------------------------------------------
PHPCBF FAILED TO FIX 1 FILE
----------------------------------------------------------------------

Time: 112ms; Memory: 10MB
```
If attempted to fix using the whole Doctrine standard, the following is produced:
```
$ phpcbf --standard=Doctrine test.php 148ms

PHPCBF RESULT SUMMARY
----------------------------------------------------------------------
FILE                                                  FIXED  REMAINING
----------------------------------------------------------------------
/home/morozov/Projects/phpcs-playground/test.php      FAILED TO FIX
----------------------------------------------------------------------
A TOTAL OF 0 ERRORS WERE FIXED IN 1 FILE
----------------------------------------------------------------------
PHPCBF FAILED TO FIX 1 FILE
----------------------------------------------------------------------

Time: 471ms; Memory: 10MB
```
```php
<?php

declare(strict_types=1);

if (true) : ?>
    foo
<?php else

 
      : ?>
    bar
<?php endif; ?>
```

**Comment by kukulich** [comment]
Unfortunately we don't have any support for "templates".

**Comment by morozov** [comment]
Would you consider a patch that makes such syntax ignored or at least not failed to fix? In the end, it's a valid PHP syntax and it has its areas of application.

**Comment by kukulich** [comment]
I hope I just make the right change to ignore it in master :)

**Comment by morozov** [comment]
740684135cf9729849c2516049a61696f8fdd3ae worked for me. Thanks, @kukulich!
